docs/docs/tags.md

```diff
@@ -0,0 +1,3 @@
+# Tags index
```

Comment:
I noticed that titles of samples in the tags index, don't align with the actual titles:

<img width="275" alt="image" src="https://user-images.githubusercontent.com/11164679/194704220-b0bfe0de-6fac-4ed7-a216-828c2e2a85e7.png">

<img width="492" alt="image" src="https://user-images.githubusercontent.com/11164679/194704228-5cd68a64-b782-4e19-8855-d991d5ebe1dc.png">

This is confusing. Let's see why it's happening and if there's anything that we can do about it.

Reply:
This is because of titles in the `nav` element in the `mkdocs.yml` file. 

They've been set up as: `- 'sample title': 'sample-scripts/spo/sample.md'`
This leads to differences with the page header in some cases. Apparently, the tags list also uses the `nav`-title.

I suggest we move to sample-scripts being defined as `- sample-scripts/spo/sample.md`. This will force mkdocs to use the first page header as a navigation title. Which in most cases translates exactly to what it is right now.

We might even do this for the rest of the navigation, but I'll leave that for now.

Reply:
> I suggest we move to sample-scripts being defined as - sample-scripts/spo/sample.md. This will force mkdocs to use the first page header as a navigation title. Which in most cases translates exactly to what it is right now.

Great idea and a nice way to increase consistency. Let's create a new issue for it so that we don't lose track of it. We can leave this as-is and pick it up in the separate issue.

Reply:
I already fixed it in this PR 🫢 maybe a bit too eager. Shall I withdraw those changes?

Reply:
I've created an issue here:

https://github.com/pnp/cli-microsoft365/issues/3859

Would you think it better to move the changes I already made to sample-scripts items?